lib/matplotlib/gridspec.py

```diff
@@ -1,16 +1,12 @@
-"""
-:mod:`~matplotlib.gridspec` is a module which specifies the location
-of the subplot in the figure.
-
-    `GridSpec`
-        specifies the geometry of the grid that a subplot will be
-        placed. The number of rows and number of columns of the grid
-        need to be set. Optionally, the subplot layout parameters
-        (e.g., left, right, etc.) can be tuned.
+r"""
+:mod:`~matplotlib.gridspec` contains classes that help to layout multiple
+`.Axes` in a grid-like pattern within a figure.
 
-    `SubplotSpec`
-        specifies the location of the subplot in the given `GridSpec`.
+The `GridSpec` specifies the overall grid structure. Individual cells within
```

Comment:
The classes including their first-line description are listed in the sphinx doc here anyway. There's no need to explicitly copy them here. Instead, just briefly describe the conceptual purpose.